src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -268,44 +281,48 @@ function dhcpd_radvd_configure($verbose = false, $blacklist = array())
             }
         }
 
-        $radvdconf .= "# Generated for DHCPv6 server $dhcpv6if\n";
-        $radvdconf .= "interface {$realif} {\n";
-        $radvdconf .= "\tAdvSendAdvert on;\n";
-        $radvdconf .= sprintf("\tMinRtrAdvInterval %s;\n", !empty($dhcpv6ifconf['ramininterval']) ? $dhcpv6ifconf['ramininterval'] : '200');
-        $radvdconf .= sprintf("\tMaxRtrAdvInterval %s;\n", !empty($dhcpv6ifconf['ramaxinterval']) ? $dhcpv6ifconf['ramaxinterval'] : '600');
+        $rtadvdconf .= "# Generated for DHCPv6 server $dhcpv6if\n";
+
+        $rtadvd_interface_list[] = $realif;
+        $rtadvdconf .= "{$realif}:\x5c\n";
+        // $radvdconf .= "\tAdvSendAdvert on;\n"; // No equivemlent?  
```

Comment:
This line should be removed, AdvSendAdvert on just specifies that radvd is periodically sending adverts.
Rtadvd always does this when an interface is specified and its running.